pages/Product/components/Content/index.jsx

```diff
@@ -31,93 +38,106 @@ class ProductContent extends Component {
     super(props);
 
     this.state = {
-      variantId: null,
+      productId: props.isBaseProduct === true ? props.id : null,
+      variantId: props.isBaseProduct === false ? props.id : null,
     };
   }
 
   /**
    * @param {Object} nextProps The next component props.
-   * @param {Object} nextState The next state.
-   * @return {boolean}
    */
-  shouldComponentUpdate(nextProps, nextState) {
-    if (this.state.variantId !== nextState.variantId) return true;
-    if (this.props.name !== nextProps.name) return true;
-    return false;
+  componentWillReceiveProps(nextProps) {
+    const isBaseProduct = nextProps.isBaseProduct === true;
```

Comment:
Please put these checks in brackets. For readability.

Reply:
Readable